pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/extension/RemoteMediaClient.kt

```diff
@@ -82,6 +83,13 @@ internal fun RemoteMediaClient.getTracks(): Tracks {
     }
 }
 
+/**
+ * MediaStatus.playbackRate return
+ * - 0 if it is paused.
+ * - Negative value if playing backward.
+ * - Positive value if playing normally.
+ */
 internal fun RemoteMediaClient.getPlaybackRate(): Float {
-    return mediaStatus?.playbackRate?.toFloat() ?: PlaybackParameters.DEFAULT.speed
+    val playbackRate = mediaStatus?.playbackRate?.toFloat().takeUnless { it == 0f }
+    return playbackRate?.absoluteValue ?: PlaybackParameters.DEFAULT.speed
```

Comment:
Is it ok to pretend that a playback rate of -3 is 3? Maybe we should just ensure that the playback rate is at least 1?

```suggestion
    return mediaStatus?.playbackRate?.toFloat()
        ?.coerceAtLeast(1f)
        ?: PlaybackParameters.DEFAULT.speed
```